src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const jwtService = require('../services/jwtService.js');
+const { ApiError } = require('../exceptions/ApiError.js');
+
+function authMiddleware(req, res, next) {
+  const authHeader = req.headers['authorization'];
+
+  if (!authHeader) {
+    throw ApiError.Unauthorized();
+  }
+
+  const accessToken = authHeader.split(' ')[1];
+
+  if (!accessToken) {
+    throw ApiError.Unauthorized();
+  }
+
+  const userData = jwtService.validateAccessToken(accessToken);
+
+  if (!userData) {
+    throw ApiError.Unauthorized();
+  }
+
+  next();
```

Comment:
The user data obtained after validating the access token is not attached to the request object. This will make it hard for subsequent middleware or request handlers to gain access to this user data.